draft-ietf-tls-tls13.md

```diff
@@ -2957,8 +2956,17 @@ the responses).
 
 
 ### Key and IV Update {#key-update}
+     enum { update_not_requested(0), update_requested(1), (255)
```

Comment:
I agree. And since I'm on the topic, I think that in any case where you're not supposed to explicitly receive extensible code points, an unknown enum is "decode_error"